website/layouts/partials/featured-image.html

```diff
@@ -12,13 +12,19 @@
 {{ end }}
 </figure>
 {{ else }}
+{{/* If this is a video page, show a preview image */}}
 {{ if $p.Params.youtubeID }}
-{{ $image := printf "https://i.ytimg.com/vi/%s/mqdefault.jpg" $p.Params.youtubeID }} 
+{{ $image := cond (isset $p.Params "image") $p.Params.image $.Site.Params.image }}
+{{/* If `image` param exists, use that in the card. */}}
+{{/* If it doesn't, use the video's medium quality thumbnail if `youtubeID` parameter exists. */}}
```

Comment:
Why are these lines needed? Aren't they redundant since you're doing the same `if` statement (as a `with`) and `$image` assignment below?

Reply:
Hm, right, it looks weird to me too now. Gonna clean it up, thanks!

Reply:
Should be good now @cjyabraham